core/org.wso2.carbon.nextgen.config/src/main/java/org/wso2/carbon/nextgen/config/ConfigConstants.java

```diff
@@ -32,5 +32,8 @@
 
     public static final String KEY_VALUE_SEPERATOR = "=";
     public static final String ENCRYPT_SECRETS = "encryptSecrets";
+    public static final String OVERRIDE_CONFIGURATION_ALWAYS = "overrideConfigurationAlways";
```

Comment:
shall we have this as "forceConfigUpdate"? Always gives a feeling that this would set to override always in subsequent runs even without the flag.